app/components/quiz/QuizQuestion.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import AppStore from 'store/appStore';
+import QuizAnswer from 'components/quiz/QuizAnswer.js';
+import QuizContext from 'components/quiz/QuizContext.js';
+
+export default React.createClass({
+  displayName: 'QuizQuestion',
+
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+
+  propTypes: {
+    question: React.PropTypes.object.isRequired,
+    onUpdate: React.PropTypes.func.isRequired
+  },
+
+  onChange: function(e) {
+    // doesn't handle checkboxes
```

Comment:
oh - it only handles how to update the `<Quiz>`'s answer if you're using radio buttons. commit coming with checkbox support